contents/blog/send-love-to-open-source.md

```diff
@@ -0,0 +1,55 @@
+---
+date: 2022-02-08
+title: Send love to open-source projects on Valentine's Day
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/send-love-to-open-source.png
+featuredImageType: full
+author: ["andy-vandervell"]
+categories: ["Open source"]
+---
+
+Valentine's Day is coming up and while PostHoggers around the world will be using the occasion to give thanks to their loved ones, we'd also like to share our love for the open-source community that helps power PostHog. Here's how:
+
+> **For every GitHub star we receive on [the PostHog repo](https://github.com/PostHog/posthog) between now through to the end Valentine's Day, we'll donate $5 USD across the four handpicked open-source projects.**
```

Comment:
to the end 'of' Valentine's Day

And in reality we will just turn this off at some point on Tuesday, but let's potentially leave it up another day or so if it's going well?